frontend/src/components/PrivateRoute/index.jsx

```diff
@@ -2,20 +2,32 @@ import React from "react";
 import { Route, Redirect } from "react-router-dom";
 import Loader from "../Loader";
 import Navbar from "../Navbar";
+import Pending from "../../pages/Pending";
 import { connect } from "react-redux";
+import { roleEnum } from "../../utils/enums";
 
-function PrivateRoute({ component: Component, authed, showLoader, ...rest }) {
+function PrivateRoute({
+  component: Component,
+  authed,
+  role,
+  showLoader,
+  ...rest
+}) {
+  const pending = role === roleEnum.PENDING;
   return (
     <div>
-      {authed && <Navbar />}
+      {!pending && <Navbar />}
```

Comment:
This will flash the navbar before showing the login page. You should make sure that `authed` is true as well.

Reply:
Added `authed && !pending && <Navbar />` 👍